src/elements/InputButton.vue

```diff
@@ -1,11 +1,16 @@
 <template>
-  <button :type="!!type ? type : false"
-    :class="['lux-button', variation, size, {'lux-expanded' : (block==true) }]"
+  <button
+    :type="!!type ? type : false"
```

Comment:
Should we add a `name` prop for buttons too?

Reply:
We are not using `<input type="button">` so I don't think we need this. I don't think you can pass a name attribute in `<button>`. Usually the button is just a trigger for submission of other form elements that contain values to be passed.